chat-ui/src/Message.tsx

```diff
@@ -30,19 +30,21 @@ export function Avatar({ role }: { role: Role }) {
           </div>
         </div>
       );
-    case "system":
-      return (
-        <div
-          className={`${styles.message_avatar} ${styles.message_avatar_system}`}
-        >
-          <Icon
-            className={styles.message_avatar_icon}
-            // glyph="Settings"
-            glyph="Megaphone"
-            color="#ffffff"
-          />
-        </div>
-      );
+    // TODO: i think this can be removed b/c we never expose the system prompt
```

Comment:
System message isn't necessarily just the prompt here - it could also be moderation/error messages.

Reply:
in that case, i don't think we should call it `system` since that's what we call the system prompt in the chatbot..could get a little confusing. 

think this should be work for future PR